alerts/openshift-container-storage-operator/CephMdsCacheUsageHigh.md

```diff
@@ -0,0 +1,36 @@
+# MDSCacheUsageHigh
+
+## Meaning
+
+Ceph MDS cache usage for the MDS daemon has exceeded above 95% of the requested value.
+
+## Impact
+
+The MDS tries to stay under a reservation of the `mds_cache_memory_limit` by
+trimming unused metadata in its cache and recalling cached items in the client
+caches. It is possible for the MDS to exceed this limit due to slow recall from
+clients. If the MDS cannot keep its cache under the target threshold, that is,
+`mds_health_cache_threshold` (150%), the MDS will send a health alert to the Monitors
+indicating the cache is too large.
+
+
+## Diagnosis
+
+Memory resources for the MDS pods should be updated in the ocs-storageCluster in
+order to increase the `mds_cache_memory_limit`. For example, run the following command
+to set the memory of MDS pods to 8GB
+
+```bash
+oc patch -n openshift-storage storagecluster ocs-storagecluster \
+    --type merge \
+    --patch '{"spec": {"resources": {"mds": {"limits": {"memory": "8Gi"},"requests": {"memory": "8Gi"}}}}}' 
+```
+
+**Note**: ODF sets `mds_cache_memory_limit` to half of the MDS pod memory request/limit.
+So if the memory is set to 8GB using above command, then the operator will set the
+mds cache memory limit to 4GB
```

Comment:
This should be in mitigation as it explains how to fix the problem and not how to identify the problem.
Maybe the diagnosis should be to verify that there is some error indicating the cache is too large (maybe ceph health details ?).

Reply:
Agree, with Umanga. Please include in the diagnosis section commands/tips/user interface clues to see how the MDS pods are using too much memory.

The patch sos command and explanation must be in the "mitigation" section.